content/en/docs/installation/guide/operator-k3s.md

```diff
@@ -0,0 +1,297 @@
+
+Before editing:
+
+```yaml
+kind: SpinnakerService
+metadata:
+  name: spinnaker
+spec:
+  # spec.spinnakerConfig - This section is how to specify configuration spinnaker
+  spinnakerConfig:
+    # spec.spinnakerConfig.config - This section contains the contents of a deployment found in a halconfig .deploymentConfigurations[0]
+    config:
+      version: 2.15.1   # the version of Spinnaker to be deployed
+      persistentStorage:
+        persistentStoreType: s3
+        s3:
+          bucket: mybucket
+          rootFolder: front50
+```
+
+For example, after editing if you named your bucket `my-s3-bucket` and want to install Armory 2.21.1:
+
+```yaml
+kind: SpinnakerService
+metadata:
+  name: spinnaker
+spec:
+  # spec.spinnakerConfig - This section is how to specify configuration spinnaker
+  spinnakerConfig:
+    # spec.spinnakerConfig.config - This section contains the contents of a deployment found in a halconfig .deploymentConfigurations[0]
+    config:
+      version: 2.21.1   # the version of Spinnaker to be deployed
+      persistentStorage:
+        persistentStoreType: s3
+        s3:
+          bucket: my-s3-bucket
+          rootFolder: front50
+```
+
+### Modifications for running on K3s
+
+Create a `spec.spinnakerConfig.config.security` section like the example below, replacing `<your-vm-ip>` with the public IP address of your EC2 instance. The `security` section is at the same level as the `persistentStorage` section.
+
+```yaml
+kind: SpinnakerService
+metadata:
+  name: spinnaker
+spec:
+  # spec.spinnakerConfig - This section is how to specify configuration spinnaker
+  spinnakerConfig:
+    # spec.spinnakerConfig.config - This section contains the contents of a deployment found in a halconfig .deploymentConfigurations[0]
+    config:
+      version: 2.21.1   # the version of Spinnaker to be deployed
+      persistentStorage:
+        persistentStoreType: s3
+        s3:
+          bucket: my-s3-bucket
+          rootFolder: front50
+      security:
+        apiSecurity: # Gate
+          overrideBaseUrl: <your-vm-ip>:8084
+        uiSecurity: # Deck
+          overrideBaseUrl: <your-vm-ip>:9000
+```
+
+Find the `expose.service.overrides` section at the bottom of the file. Add configuration for Deck and Gate.
+
+```yaml
+expose:
+  type: service  # Kubernetes LoadBalancer type (service/ingress), note: only "service" is supported for now
+  service:
+    type: LoadBalancer
+    annotations:
+      service.beta.kubernetes.io/aws-load-balancer-backend-protocol: http
+    # provide an override to the exposing KubernetesService
+  overrides:
+    deck:
+      publicPort: 9000
+    gate:
+      publicPort: 8084
+```
+
+Spacing is very important in YAML files. Make sure the spacing is correct in the `SpinnakerService.yml` file . Also, ensure there are no tabs instead of spaces in the file. Incorrect spacing or tabs will cause errors when you install Spinnaker.
```

Comment:
```suggestion
Spacing is very important in YAML files. Make sure that the spacing is correct in the `SpinnakerService.yml` file and that there are no tabs instead of spaces. Incorrect spacing or tabs cause errors when you install Spinnaker.